### PR TITLE
Clean up cases in bihar domain

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?166020#923418

@snopoke

### CODE REVIEW

File: custom/bihar/management/commands/bihar_cleanup_case.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                case.owner_id = parent.owner_id
+                csv_row[4] = case.owner_id
+                should_save = True
+
+            if should_save:
+                to_save.append(case)
+                csv_file.writerow(csv_row)
+
+            if len(to_save) > 25:
+                CommCareCase.get_db().bulk_save(cases)
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case.py]: F821 undefined name 'cases'

File: custom/bihar/management/commands/bihar_cleanup_case.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                to_save = []
+
+            if i % 100 == 0:
+                logger.info("{current}/{count} cases completed".format(current=i, count=len(case_ids)))
+
+        if to_save:
+            CommCareCase.get_db().bulk_save(cases)
```
Comment (dimagimon) [inline custom/bihar/management/commands/bihar_cleanup_case.py]: F821 undefined name 'cases'

File: custom/bihar/management/commands/bihar_cleanup_case.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        to_save = []
+
+        for i, doc in enumerate(iter_docs(CommCareCase.get_db(), case_ids)):
+            should_save = False
+            case = CommCareCase.wrap(doc)
+            parent = case.parent
+            csv_row = [case._id, case.type, case.type, case.owner_id, case.owner_id]
+
+            if case.type != 'task':
+                if case.user_id != MOTECH_ID:
```
Comment (emord) [inline custom/bihar/management/commands/bihar_cleanup_case.py]: Didn't see an index to search for this and domain/case type (only owner_id).

File: custom/bihar/management/commands/bihar_cleanup_case.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            if len(to_save) > 25:
+                CommCareCase.get_db().bulk_save(to_save)
+                to_save = []
+
+            if i % 100 == 0:
+                logger.info("{current}/{count} cases completed".format(current=i, count=len(case_ids)))
+
+        if to_save:
+            CommCareCase.get_db().bulk_save(to_save)
+            csv_file.writerow(csv_row)
```
Comment (snopoke) [inline custom/bihar/management/commands/bihar_cleanup_case.py]: I don't think this last csv_file write is necessary

**Pull request merged**